package.json

```diff
@@ -37,7 +37,7 @@
     "file": "^0.2.2",
     "gavel": "^0.5.3",
     "glob": "^7.0.5",
-    "html": "0.0.10",
+    "apiaryio/commonjs-html-prettyprinter": "",
```

Comment:
The thing is, for `npm install <something>` it works: https://docs.npmjs.com/cli/install

> Note: if you do not include the @-symbol on your scope name, npm will interpret this as a GitHub repository instead, see below. Scopes names must also be followed by a slash.

```
npm install githubname/reponame
```

But it doesn't work for `package.json`: https://docs.npmjs.com/files/package.json#github-urls